src/App/Program.fs

```diff
@@ -205,6 +208,24 @@ type MissionOptions
              Default = false)>]
     member self.UnevenSched = unevenSched
 
+    [<Option("require-node-labels",
+             HelpText = "Only run on nodes with matching `key:value` labels",
+             Required = false,
+             Default = false)>]
+    member self.RequireNodeLabels = requireNodeLabels
+
+    [<Option("avoid-node-labels",
+             HelpText = "Do not run on nodes with matching `key:value` labels",
+             Required = false,
+             Default = false)>]
+    member self.AvoidNodeLabels = avoidNodeLabels
+
+    [<Option("tolerate-node-taints",
+             HelpText = "Allow running on nodes with matching taints",
+             Required = false,
+             Default = false)>]
+    member self.TolerateNodeTaints = tolerateNodeTaints
```

Comment:
Do we allow those options to be repeated? It's probably good enough if we don't but if it's easy to allow multiple values it could be useful at some point (for example `--require-node-labels os:linux --require-node-labels size:large`)

Reply:
They are multi-argument options, so are specified by `--require-node-labels os:linux size:large`, but yes.